src/index.js

```diff
@@ -14,7 +14,15 @@ function linear(wait: Wait): Function {
     const timeouts: Timeouts = {},
         delays: Array<string> = Object.keys(wait);
 
-    return function(...args): void {
+    function cancel(): void {
+        for (const timeout: string in timeouts) {
+            if (timeouts.hasOwnProperty(timeout)) {
```

Comment:
Use of `Object.values` can save you checking `hasOwnProperty`:
```
Object.values(timeouts).forEach(clearTimeout)
```

Reply:
Nice one. Added.